### ISSUE TITLE
Checkbox group: On accessing the check box as highlighted in red, label ‘My favorites Fruits’ is not read out. I.e. label is not associated to their corresponding UI’s. 

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3824]
Description: On accessing the check box as highlighted in red, label ‘My favorites Fruits’ is not read out. I.e. label is not associated to their corresponding UI’s. 

Expected: Corresponding label must be read out, when each & every UI is accessed.

Screenshot: 

![image](https://user-images.githubusercontent.com/32538291/98773716-83d7cf00-240f-11eb-846a-7c414831b5a3.png)

**Comment by DeepakSap14** [comment]
https://user-images.githubusercontent.com/57661487/102957417-5f880b80-4500-11eb-8e25-1ccbbdce565d.mov

I am not able to reproduce this.
I have tested in chrome on Mac.
